### DISCUSSION TITLE
[v3] POST http://127.0.0.1:8000/livewire/update 404 (Not Found) and No query results for model in Delete process

### DISCUSSION THREAD

**Discussion by gunnery34** [livewire/livewire#8663] [7. Bugs (archived)]
### Livewire version

v3.4.6

### Laravel version

v11.16.0

### Which PHP version are you using?

PHP 8.2

### Steps To Reproduce

I tried using Livewire with Volt
Initially everything runs normally without any problems like wire:navigate in SPA for view, add, edit and update.
But on delete I get a strange thing

My code for delete is
```php
public function deleteOption($optionId): void
{
    \Log::info('Deleting option with ID: ' . $optionId);
    $option = Option::find($optionId);
    if ($option) {
        $this->deleteData($option);
    } else {
        \Log::error('Option not found with ID: ' . $optionId);
        $this->error('Option not found!');
    }
}

protected function deleteData(Option $option): void
{
    \Log::info('Deleting data for option with ID: ' . $option->id);
    $option->delete();
    $this->success('Option deleted!');
}
```

I checked on `storage/logs/laravel.log` I got the Option Id correctly
![fgehrtjuykiutjyrh](https://github.com/user-attachments/assets/a1214bb9-7c1c-4bad-bbee-b7c3aa4789ae)

But when I try to delete I get a 404 display as shown below
![gtrhytjyikujtyhr](https://github.com/user-attachments/assets/048ead53-3327-4eb8-9daa-7da4ee4c0637)

I get a `POST error http://127.0.0.1:8000/livewire/update 404 (Not Found)` on the console as shown below
![dfeghtujyiyj6ety5hwtr4g](https://github.com/user-attachments/assets/6bc7947e-1bfe-4e82-8250-8e729f2eed48)

Then on the Debugbar I get an error `No query results for model [App\Models\Option]` as shown below
![rtghyjukjtyhrt](https://github.com/user-attachments/assets/325f3dcc-9778-474f-b6be-2e8c20193c96)

Does this problem seem to be with Livewire/Volt to me
Can anyone help me with this problem?
Thank you

### Repository or Wirebox URL

-

### Please confirm (incomplete submissions will not be addressed)

- [X] I have provided easy and step-by-step instructions to reproduce the bug.
- [X] I have provided code samples as text and NOT images.
- [X] I understand my bug report will be removed if I haven't met the criteria above.

**Comment by shazimali** [comment]
did you get any solution, i am getting the same problem

**Reply by gunnery34** [reply]
Sorry, I haven't found a way to do this yet
However, I managed to solve the issue by downgrading my Laravel version from 11.x to 10.x
